canonical/ubuntu.com
# Initial form template

## Done

Created a template to make forms dynamic

REMOVE FORM FROM BASE INDEX AFTER TESTING

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes
- Check that the form template is rendered correctly and submits data as expected

## Issue / Card
[WD-6181](https://warthogs.atlassian.net/browse/WD-6181)


[WD-6181]: https://warthogs.atlassian.net/browse/WD-6181?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
